test/functional/developer_portal/admin/messages/outbox_controller_test.rb

```diff
@@ -30,6 +30,26 @@ def setup
     assert_equal "web", msg.origin
   end
 
+   test "will not create messages without subject and body" do
```

Comment:
I think we could test all cases, trying to send messages without subject and body but also without message or body.

Reply:
Okay you mean two more cases

1. No body but subject is present
2. No subject but body is present 

Yes I can add test for both cases, but the logic will only allow when we keep both body and subject

Reply:
@lvillen Added both cases

Reply:
Yes! That was what I meant!